public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -455,7 +455,7 @@ function buildMetricsQuery(query: TraceToMetricQuery, tags: Array<KeyValue<strin
     }, [] as string[]);
 
     const labelsQuery = labels?.join(', ');
-    expr = expr.replace('$__tags', labelsQuery);
```

Comment:
hmm we should probably use the template service to do the replacement for us mainly so the syntax for variable interpolation is the same across grafana.